start-finalSetup04ServerProperties

```diff
@@ -77,6 +77,18 @@ function customizeServerProps {
   setServerProp "resource-pack" "$RESOURCE_PACK"
   setServerProp "resource-pack-sha1" "$RESOURCE_PACK_SHA1"
   setServerProp "player-idle-timeout" "$PLAYER_IDLE_TIMEOUT"
+  setServerProp "broadcast-console-to-ops" "$BROADCAST_CONSOLE_TO_OPS"
+  setServerProp "broadcast-rcon-to-ops" "$BROADCAST_RCON_TO_OPS"
+  setServerProp "enable-jmx-monitoring" "$ENABLE_JMX"
```

Comment:
That's cool it coincided with the `ENABLE_JMX` I had before. Makes this option even more useful.

https://github.com/itzg/docker-minecraft-server/blob/f5dde77efe253e7ac668acb721f6257db32f818d/start-minecraftFinalSetup#L103-L116